owl/EASE-PROC.owl

```diff
@@ -273,12 +293,6 @@ A soft slab of clay deforming under its own weight on Earth would still count as
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-PROC.owl#Destruction">
         <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-PROC.owl#ProcessType"/>
-        <rdfs:subClassOf>
-            <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasDestroyed"/>
```

Comment:
Similarly,

Destruction subclassOf classifies only (hasDestroyed some PhysicalObject)

Reply:
> The hasCreated, hasDestroyed axioms in the definition of process types Creation and Destruction should be corrected and kept I think. We may need to reason with entity creation or destruction in the cooking domain.

The thing is that then we would need to add a property for *all* the roles. As there is a role *DestroyedObject*, I think. I would like to keep redundancy low, and roles being concepts rather then properties.

Unfortunately, there is no fitting property for linking anything other then *Task* to roles (using isTaskOf). But I would rather vote for at some point start to axiomatize *EventTypes* with their roles and parameters -- this is not done yet at all.

Reply:
there is also a ticket about adding role/parameter axioms: https://github.com/ease-crc/ease_ontology/issues/24

Reply:
Ok, then how about

Creation subclassof classifies only ('has participant' some (PhysicalObject and 'is classified by' some CreatedObject)

Destruction subclassof classifies only ('has participant' some (PhysicalObject and 'is classified by' some DestroyedObject)

Reply:
but wouldn't that mean that the object is *always* classified as *CreatedObject*? Also, since the *Creation* concept has the role *CreatedObject* (at least in the future at some point), I think this would again be redundant. It should be enough to state that *Creation* is a process of the role *CreatedObject* and that this role only classifies *PhysicalObject*

Reply:
To answer the question, no. Here's a model of the axiom above for Creation:

let C be a Creation individual, therefore a Process Type. It classifies P, an individual of type Process. P hasParticipant O1 and O2, both of which are individuals of type PhysicalObject. O1 is classified by CO, an individual of type CreatedObject. O2 is NOT classified by any role.

As to the future of Creation, why not make the future now :) The axioms I suggested do exactly the thing you require: they link Creation and CreatedObjects as classifiers for Processes and some participants in them.

Reply:
As of the future is now: this is not the scope of the pull request. The axiomatization of process types with tasks and roles probably should be done in a separate effort.

Reply:
> O1 is classified by CO, an individual of type CreatedObject.

"O1 is classified by CO, an individual of type CreatedObject." -- no. It is classified as such only in the scope of the occurrence of a *Creation* process.

Reply:
Ok. Then we'll look into this after the holidays.